pom.xml

```diff
@@ -330,6 +330,7 @@
                 <artifactId>maven-javadoc-plugin</artifactId>
                 <version>2.9.1</version>
                 <configuration>
+                    <javadocExecutable>${java.home}/../bin/javadoc</javadocExecutable>
```

Comment:
Not sure whether to leave this there or not. Currently if you just clone the code and set the JDK path the build fails. This fixes it although it doesn't look pretty, I'd prefer adding a note on how to set the javadoc path in the docs.